src/style.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --search-icon: url(./images/Search.svg);
+  --search-form-font: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf"),
+    url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+
+body {
+  font-family: var(--search-form-font);
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__form__input {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
61, 78, 97 duplicated color, so make sense to move it to the variables
check other places